jenkins_job_wrecker/modules/publishers.py

```diff
@@ -148,10 +152,90 @@ def buildtrigger(top, parent):
+                                for element in subconf:
+                                    if element.tag == "name":
+                                        project["node-label-name"] = element.text
+                                    elif element.tag == "nodeLabel":
+                                        project["node-label"] = element.text
+                                    else:
+                                        raise NotImplementedError("cannot handle NodeLabelBuildParameter "
+                                                                  "sub-element %s " % element.tag)
+                            elif subconf.tag == 'hudson.plugins.parameterizedtrigger.CurrentBuildParameters':
+                                current_parameters_exist = True
+                            elif subconf.tag == 'hudson.plugins.parameterizedtrigger.NodeParameters':
+                                node_parameter_exist = True
+                            elif subconf.tag == 'hudson.plugins.parameterizedtrigger.SubversionRevisionBuildParameters':
+                                project["svn-revision"] = True
+                                for element in subconf:
+                                    if element.tag == 'includeUpstreamParameters':
+                                        project["include-upstream"] = get_bool(element.text)
+                                    else:
+                                        raise NotImplementedError("cannot handle SubversionRevisionBuildParameters "
+                                                                  "sub-element %s " % element.tag)
+                            elif subconf.tag == 'hudson.plugins.parameterizedtrigger.BooleanParameters':
+                                for config in subconf:
+                                    if config.tag == "configs":
+                                        boolean_config = {}
+                                        for bool_param_config in config:
+                                            if bool_param_config.tag == \
+                                                    "hudson.plugins.parameterizedtrigger.BooleanParameterConfig":
+                                                if len(bool_param_config) == 2 and bool_param_config[0].tag == "name" \
+                                                        and bool_param_config[1].tag == "value":
+                                                    boolean_config[bool_param_config[0].text] = \
+                                                        get_bool(bool_param_config[1].text)
+                                            else:
+                                                raise NotImplementedError("cannot handle BooleanParameters"
+                                                                          "sub-element %s " % bool_param_config.tag)
+                                        project["boolean-parameters"] = boolean_config
```

Comment:
What happens when you have multiple boolean parameters?

Reply:
boolean parameters are held in xml under `hudson.plugins.parameterizedtrigger.BooleanParameters > configs` as `hudson.plugins.parameterizedtrigger.BooleanParameterConfig`, there can only be `name` and `value` under `hudson.plugins.parameterizedtrigger.BooleanParameterConfig`. If there are mutliple boolean parameters, it will go through 
```
for bool_param_config in config:
                                            if bool_param_config.tag == \
                                                    "hudson.plugins.parameterizedtrigger.BooleanParameterConfig":
```

Which will add the key-value inside the `boolean_config` dict. Hope this explains.